klippa/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "klippa"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+font-types = { path = "../font-types" }
+read-fonts = { path = "../read-fonts" }
+thiserror = "1.0.58"
+write-fonts = { path = "../write-fonts" }
```

Add the `read` feature here to get the `write_fonts::read` module I mention above:


```suggestion
write-fonts = { path = "../write-fonts", features = ["read"] }
```